src/views/LoadManagementConfig.vue

```diff
@@ -239,6 +245,33 @@
 								(Zwischen-)Zählers.
 							</template>
 						</openwb-base-number-input>
+						<openwb-base-number-input
+							title="Maximale Leistung im Fehlerfall"
+							:min="1"
```

Comment:
```suggestion
							:min="0"
```
Mit dem Wert "0" sollte jegliche Ladung komplett unterbunden werden können (aktueller Standard).

Reply:
umgesetzt